test/e2e/tests/test_function.py

```diff
@@ -395,7 +395,7 @@ def test_function_package_type_image_with_signing_config(self, lambda_client):
 
         time.sleep(UPDATE_WAIT_AFTER_SECONDS)
 
-        cr = k8s.wait_resource_consumed_by_controller(ref)
+        # cr = k8s.wait_resource_consumed_by_controller(ref)
```

Comment:
Is this okay to be commented out?

Reply:
Yes, this line of code waits for resource to be consumed after update and store the new status in "cr". We are doing the same thing in [Line 407](https://github.com/aws-controllers-k8s/lambda-controller/blob/main/test/e2e/tests/test_function.py#L407) too. The idea is when the first update is made, we wait and let the controller consume the resource, save the new status in `cr` and use it for second update. Either of the place for this line of code would work fine. I just chose to do it right before moving for the second update. Keeping both increases the time of execution and lead to timeout while running the tests.

Reply:
I see. In that case, we should delete one instead of commenting it

Reply:
Done